setup_cfg_fmt.py

```diff
@@ -204,10 +208,8 @@ def _python_requires(
 
     if minimum is None:
         return None
-    elif min_py_version > minimum:
-        return _format_python_requires(min_py_version, excluded)
-    else:
-        return _format_python_requires(minimum, excluded)
```

Comment:
I think the first part of this if statement is still useful? like I kinda want to be able to just change `setup-cfg-fmt` and have it update the value in all of my projects (which is what it does now) -- I think all that's needed is to somehow keep the minimum supported python version around -- maybe throw min and max at the top of the file and use that to decide here?

Reply:
So doesn't that make the flag redundant? If you want to increase the floor in your project you'd have to change both `python_requires` and `--min-py-version`? Whereas this implementation lets `--min-py-version` be the source of truth

I think now that we this tool doesn't support py3 anymore then it makes sense for `--min-py-version` to be the single place that configures the version floor, and `python_requires` + classifiers will be changed based on its value

Reply:
Put another way, is `--min-py-version` a useful flag to have if we want `python_requires` to be the source of truth? I could see why `--min-py3-version` was useful but with the reasoning you gave at the start of this thread, it makes me wonder if `--min-py-version` isn't needed anymore

Reply:
I think I want:
- specify nothing: minimum is `max(3.7, python_requires minimum, tox minimum, ...)`
- specify `--min-py-version`: minimum is `min_py_version`

that way implicitly it'll get upgraded as setup-cfg-fmt gets updated *unless* you explicitly want

Reply:
OK sg. Do you think the `ImplicitlySetVersion` is an OK approach for detecting if nothing was specified?

Reply:
`None` is probably simpler -- either works up to you

Reply:
OK. And we should write `python_requires` always?

Reply:
yep!